ads/kixer.js

```diff
@@ -45,6 +46,17 @@ export function kixer(global, data) {
   };
   d.addEventListener('load', kxload, false);
 
+  const unlisten = global.context.observeIntersection(function(changes) {
+    changes.forEach(function(c) {
+      if (c.intersectionRect.height > 0) {
+        if (typeof __kx_viewability.process_locked === 'function') {
```

Comment:
Makes sense. I tried the loadPromise suggestion but we use our own 'load' event to know when the ad has loaded, so I moved the viewability check to the listener.